spaces/domains.json

```
@@ -129,5 +129,6 @@
   "vote.1-b.tc":"1btc.eth",
   "poll.santiment.net": "santiment.eth",
   "voting-bmxx.multiplier.finance": "mcl-multiplierfinance.eth",
-  "vote.cndy.finance": "candyfinance.eth"
+  "vote.cndy.finance": "candyfinance.eth",
+  "gov.riseprotocol.io': "riseprotocol.eth"
```

You must replace the single quote with a double quote.
```suggestion
  "gov.riseprotocol.io": "riseprotocol.eth"
```